cycledash/api/comments.py

```diff
@@ -174,8 +177,30 @@ def get_last_comments(n=5):
         q = select(cols.values()).order_by(
             desc(cols.created)).limit(n)
         comments = [camelcase_dict(dict(c)) for c in con.execute(q).fetchall()]
-    return epochify_comments(comments)
+    return userify_comments(epochify_comments(comments))
 
+def userify_comments(comments):
+    """Given comments with userIds, attaches the relevant user
+    info to the comments
+    """
+    for comment in comments:
+        userify_comment(comment)
+    return comments
+
+def userify_comment(comment):
+    """Given a comment with userId, attaches the relevant user
+    info to the comment
+    """
+    if 'userId' in comment:
+        with tables(db.engine, 'users') as (con, users):
+            q = select(users.c).where(users.c.id == comment['userId'])
```

Comment:
`q = select([users.c.username, users.c.email, users.c.id])...`; generally better to whitelist than blacklist (which also lets you get rid of "desentisize_user" (if keeping it, spelling -> desensitize).